bin/flush_cache

```diff
@@ -4,29 +4,11 @@ require_relative '../deployment'
 require 'cdo/rake_utils'
 require 'cdo/aws/cloudfront'
 
-require 'sshkit'
-SSHKit::Backend::Netssh.configure {|ssh| ssh.ssh_options = {paranoid: false}}
-
 unless CDO.daemon
   CDO.log.error 'This script must be run on daemon'
```

Comment:
is this still true?

Reply:
We were only using `sshkit` to connect to each web application server EC2 Instance and then invoke commands on the instance of Varnish installed on those servers, so we don't need to load and initialize it anymore.